lib/ap/internal/fcall.h

```diff
@@ -30,13 +30,13 @@ enum
 struct	Fcall
 {
 	char	type;
-	short	fid;
-	unsigned short	tag;
+	int16_t	fid;
```

Comment:
It seems clearly to matter here, however. But fids should be unsigned. This is weird.